source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -0,0 +1,451 @@
+    ViConstString srcTerminal,
+    ViConstString destTerminal,
+    ViConstString syncClock,
+    ViInt32 invert,
+    ViInt32 updateEdge,
+    ViStatus* viStatusOut)
+  {
+    ::grpc::ClientContext clientContext;
+    nisync::ConnectTrigTerminalsRequest request;
+    nisync::ConnectTrigTerminalsResponse response;
+    request.set_src_terminal(srcTerminal);
+    request.set_dest_terminal(destTerminal);
+    request.set_sync_clock(syncClock);
+    request.set_invert(invert);
+    request.set_update_edge(updateEdge);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->ConnectTrigTerminals(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+
+  ::grpc::Status call_DisconnectTrigTerminals(ViConstString srcTerminal, ViConstString destTerminal, ViStatus* viStatusOut)
+  {
+    ::grpc::ClientContext clientContext;
+    nisync::DisconnectTrigTerminalsRequest request;
+    nisync::DisconnectTrigTerminalsResponse response;
+    request.set_src_terminal(srcTerminal);
+    request.set_dest_terminal(destTerminal);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->DisconnectTrigTerminals(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+
+private:
+  std::shared_ptr<::grpc::Channel> channel_;
+  std::unique_ptr<::nidevice_grpc::Session> driver_session_;
+  std::unique_ptr<nisync::NiSync::Stub> nisync_stub_;
+  std::unique_ptr<nidevice_grpc::SessionRepository> session_repository_;
+  std::unique_ptr<nisync::NiSyncLibrary> nisync_library_;
+  std::unique_ptr<nisync::NiSyncService> nisync_service_;
+  std::unique_ptr<::grpc::Server> server_;
+};
+
+TEST_F(NiSyncDriverApiTest, ConnectClkTerminals_ReturnsSuccess)
+{
+  ViStatus viStatus;
+  auto srcTerminal = NISYNC_VAL_OSCILLATOR, destTerminal = NISYNC_VAL_CLKOUT;
+  auto grpcStatus = call_ConnectClkTerminals(srcTerminal, destTerminal, &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatus);
+  call_DisconnectClkTerminals(srcTerminal, destTerminal, &viStatus);
+}
+
+TEST_F(NiSyncDriverApiTest, ConnectInvalidClkTerminals_ReturnsInvalidSrcTerminal)
+{
+  ViStatus viStatus;
+  auto srcTerminal = kInvalidTerminal, destTerminal = NISYNC_VAL_CLKOUT;
+  auto grpcStatus = call_ConnectClkTerminals(srcTerminal, destTerminal, &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(kSyncInvalidSrcTerminal, viStatus);
+}
+
+TEST_F(NiSyncDriverApiTest, ConnectedClkTerminals_DisconnectClkTerminals_ReturnsSuccess)
+{
+  ViStatus viStatus;
+  auto srcTerminal = NISYNC_VAL_OSCILLATOR, destTerminal = NISYNC_VAL_CLKOUT;
+  auto grpcStatus = call_ConnectClkTerminals(srcTerminal, destTerminal, &viStatus);
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatus);
+
+  grpcStatus = call_DisconnectClkTerminals(srcTerminal, destTerminal, &viStatus);
+  EXPECT_TRUE(grpcStatus.ok());
```

Comment:
Minor: put a blank line before the asserts so that the test code blocks match the segments of the name.
Same on tests below.

Reply:
fixed